api/src/main/kotlin/edu/wgu/osmt/api/model/ApiCollectionV2.kt

```diff
@@ -45,4 +45,4 @@ class ApiCollectionV2(
             return result
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Whitespace changes should not be made on a PR against a feature branch. Diff hell for everyone.